### DISCUSSION TITLE
PSQL binary not found

### DISCUSSION THREAD

**Discussion by Jolt32** [akiver/cs-demo-manager#645] [Q&A] [answered]
but see PSQL binary not found in CS demo manager 3.0.0

**Comment by Jolt32** [comment, marked as answer]
nevermind i just system restart problem solved my bad :D

**Comment by Jolt32** [comment]
is valves demos currently broken or something i can only download certain matches

**Reply by akiver** [reply]
If you see this orange icon, it means the demo link is not working (expired or 404). 
Since the v3.0.1 there is a button to copy the demo link - you can then try it from your browser.
<img width="184" alt="Screenshot 2023-12-03 at 02 53 42" src="https://github.com/akiver/cs-demo-manager/assets/1842524/baeb1a86-0b81-46a4-9830-0e7810e82dcc">

Sometimes matchmaking demos are not uploaded or can take some time to be available.

**Reply by Jolt32** [reply]
i will try what you said thank you :)

**Reply by Jolt32** [reply]
oh yeah why is auto download start up on by default? i feel like people would only want to download certain demo, or maybe just me.

**Comment by Jolt32** [comment]
love the demo manager :) makes it easier
